Elements/Elements/ViewControllers/GameViewController.swift

```diff
@@ -39,7 +39,11 @@ extension GameViewController {
         queryLabel.text = viewModel.queryTitle
         for (index,button) in answersButtons.enumerated() {
             button.setTitle(viewModel.answers[index], for: UIControlState())
-            button.backgroundColor = UIColor.white
+            button.backgroundColor = UIColor.cyan
+            button.layer.cornerRadius = 5
+            button.layer.borderWidth = 1
+            button.layer.borderColor = UIColor.black.cgColor
+            
```

Comment:
It's fine but take a look at https://github.com/rhodgkins/UIView-IBInspectable/blob/master/IBInspectables.swift
Read about IBInspectable - you can set values like the cornerRadius, borderWidth and borderColor from Interface Builder! Read also about extension for Swift classes.
Finally create an extension for UIView (UIButton inherits from UIView) like in the example above and set those values for every single button in the Interface Builder.
Your solution is absolutely fine. But do this exercise just for practise purposes.